packages/graph-explorer-proxy-server/node-server.js

```diff
@@ -209,8 +212,9 @@ async function fetchData(res, next, url, options, isIamEnabled, region) {
 
     const isIamEnabled = !!req.headers["aws-neptune-region"];
     const region = isIamEnabled ? req.headers["aws-neptune-region"] : "";
+    const serviceType = isIamEnabled ? (req.headers["service-type"] ?? "neptune-db") : "";
```

Comment:
nit: extract "neptune-db" as a constant - DEFAULT_SERVICE_TYPE (as it is in multiple places)